fbchat/client.py

```diff
@@ -10,8 +10,8 @@
 from .utils import *
 from .models import *
 from .graphql import *
-import time
```

Comment:
`time` is still being used in `time.sleep`, and neither `datetime` nor `calendar` is being used